tests/h/search/query_test.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import pytest
+
+from h.search import Search
+from h.search import TopLevelAnnotationsFilter
+
+
+class TestTopLevelAnnotationsFilter(object):
+
+    def test_it_filters_out_replies_but_leaves_annotations_in(self, Annotation, search):
+        annotation = Annotation(shared=True)
+        reply = Annotation(references=[annotation.id], shared=True)
+
+        result = search.run({})
+
+        assert annotation.id in result.annotation_ids
+        assert reply.id not in result.annotation_ids
+
+    @pytest.fixture
+    def search(self, pyramid_request):
+        search = Search(pyramid_request)
+        search.append_filter(TopLevelAnnotationsFilter())
+        return search
```

Comment:
My intention is that each test class for each filter will have its own `search` fixture that returns a `Search` instance with that filter added. The default filters are also present on this `Search` instance. (The test classes for the default filters won't need to append them since they're already appended by default. By not appending them, they will handily test the appending-by-default too.)